lib/commands/os/configure.ts

```diff
@@ -488,3 +488,39 @@ function camelifyConfigOptions(options: FlagsDef): { [key: string]: any } {
 		return key;
 	});
 }
+
+/**
+ * Returns the boot partition number of the given image.
+ * @param imagePath Local filesystem path to a balenaOS image file
+ */
+async function getBootPartition(imagePath: string): Promise<number> {
+	const filedisk = require('file-disk');
+	const partitioninfo = require('partitioninfo');
+
+	const partitionnumber = await filedisk.withOpenFile(
+		imagePath,
+		'r',
+		async (handle: any) => {
```

Comment:
Hopefully this won't have to be type `: any` when `require` is replaced with `await import`. 🤞 
(Try simply deleting `: any` and not replacing it with anything -- sometimes `tsc` is smart enough to figure out the type by itself.)

Reply:
Just like you said, `tsc` is smart enough. 👍🏼